packages/xo-web/src/icons.scss

```diff
@@ -181,6 +181,10 @@
     @extend .fa;
     @extend .fa-play;
   }
+  &-freeze {
+    @extend .fa;
+    @extend .fa-snowflake-o;
+  }
```

Comment:
For the icon, what do you think of [lightbulb](https://fontawesome.com/v4/icon/lightbulb-o)?

Reply:
I wanted to show that the content of the host (ie the VMs) were "frozen"

Reply:
@olivierlambert What do you think?

Reply:
I understand the concept behind the snowflake icon and I don't have a better idea ATM :+1: